zng/docs/spec.md

```diff
@@ -367,7 +396,10 @@ length in bytes of the type ID.
 ------------------------
 ```
 
-A typed value with a `value` of length N and the type indicated
+It is an error for a value to reference a type ID that has not been previously
+defined by a typedef scoped to the stream in which the value appears.
```

Comment:
I moved this up from the TZNG section, as it reads as if it's about ZNG in general.